src/dataset-operator/pkg/controller/datasetinternal/datasetinternal_controller.go

```diff
@@ -841,3 +880,36 @@ func createPVCforObjectStorage(cr *comv1alpha1.DatasetInternal, rc *ReconcileDat
 	return result, err
 
 }
+
+func getBooleanStringForKeyInMap(key string,mapString map[string]string) string {
```

Comment:
Handy helper function! Proposal: add another parameter which will be the default value to return if the map does not contain the key or an error occurs. In this way, if we need to return "true" in the future we will employ the same function.